package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "avatar-twitch-bot",
+    "version": "1.0.0",
+    "description": "",
+    "engines": {
+        "node": ">=14",
+        "npm": ">=6"
+    },
+    "scripts": {
+        "build": "node_modules/.bin/tsc",
+        "start": "node ./dist/main",
+        "lint": "node_modules/.bin/prettier --check . && node_modules/.bin/eslint \"src/**/*.ts\"",
+        "prettier": "node_modules/.bin/prettier --write .",
+        "prepare": "husky install"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/paifgx/avatar-twitch-bot.git"
+    },
+    "author": "",
+    "license": "MIT",
+    "bugs": {
+        "url": "https://github.com/paifgx/avatar-twitch-bot/issues"
+    },
+    "homepage": "https://github.com/paifgx/avatar-twitch-bot#readme",
+    "dependencies": {
+        "tmi.js": "~1.8.3"
+    },
+    "devDependencies": {
+        "@types/node": "~15.12.1",
+        "@types/tmi.js": "~1.7.1",
+        "@typescript-eslint/eslint-plugin": "~4.26.0",
+        "@typescript-eslint/parser": "~4.26.0",
+        "dotenv": "~10.0.0",
```

Comment:
would dotenv not rather be a dependency instead of a devDependency ? aka it is necessary at runtime too and not just at compile time.

Reply:
That is a good point. I would actually check that again just to be sure. But I think, since we use the TypeScript definitions from DotEnv, that the required functions are delivered / transposed here.

Reply:
Ok, if this is the case when TS is translated into JS, i am fine with this.